Tests/ML/reports/test_segmentation_report.py

```diff
@@ -31,6 +39,43 @@ def test_generate_segmentation_report(test_output_dirs: OutputFolderForTests) ->
     contents = result_html.read_text(encoding='utf-8')
     assert 'parotid_r' in contents
 
+def _test_generate_segmentation_report_with_partial_ground_truth(test_output_dirs: OutputFolderForTests) -> None:
+    """
+    The test without partial ground truth should cover more detail, here we just check that providing
+    partial ground truth results in some labels having a lower user count.
+    """
+    reports_folder = Path(__file__).parent
```

Comment:
Here and above: Please don't create new custom ways of accessing files in the test folder. It reduces maintainability. Instead, use the pre-defined `full_ml_test_data_path` or something like that.

Reply:
Not my code, but I'll fix that.